src/components/new-request/name-check/name-check.vue

```diff
@@ -12,7 +12,7 @@
       </v-col>
     </v-row>
     <v-row class="px-10 pt-6" no-gutters>
-      <v-col :class="getIsXproMras ? 'pr-8' : ''" cols="12" lg="7">
+      <v-col :class="getIsXproMras ? 'pr-8' : ''" cols="12" md="10" lg="10">
```

Comment:
You may not need the "lg" is the "md" is set.
For next time.

Reply:
kk ty, will look into this.